wiki/en/Running-a-Server.md

```diff
@@ -50,55 +49,47 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
+### 1. Registered 
 
-Your server will be listed in the Directory that clients use by default. Musicians can then discover and connect to your server.
+Your server will be listed in a Directory. By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, anyone can then discover and connect to your server. 
 
 <figure>
-	<img src="{% include img/en-screenshots/diagram-public-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Public Server">
-<figcaption>How Public Servers work</figcaption>
+	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Registered Server">
+<figcaption>How Registered Servers work</figcaption>
 </figure>
 
+### 2. Unregistered 
 
-
-### 2. Private
-
-This is the default when starting a server for the first time. Private Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not (in any server mode) let you control who can connect to a server.
-
+This is the default when starting a server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a server.
 
 <figure>
-	<img src="{% include img/en-screenshots/diagram-private-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Private Server">
-	<figcaption>How Private Servers work</figcaption>
+	<img src="{% include img/en-screenshots/diagram-unreg-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Unregistered Server">
+	<figcaption>How Unregistered Servers work</figcaption>
 </figure>
 
+### 3. Custom Directory
 
+If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
 
-### 3. Directory
-
-For when you want to run a number of Private Servers, possibly also behind a firewall or on a LAN. Examples include online events, music associations, sectional rehearsals or music lessons for schools.
-
-To view servers listed by a custom Directory Server, musicians must enter the address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by your Directory.
+To view servers listed by a Custom Directory, musicians must enter its address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by your Directory. Custom Directories otherwise work for clients in the same way as Public Directories.
```

Comment:
> client’s "Custom Directory Server" settings field
![image](https://user-images.githubusercontent.com/1549463/148685228-f12728ac-0d94-49ac-b380-25baa4c11823.png)
"Custom Directories", as we store multiple at the client.

Reply:
> They will then see a Connection Setup list
![image](https://user-images.githubusercontent.com/1549463/148685326-02c68912-39d2-4279-a16f-7e96ee044836.png)
It's just the "Connect" window, where the server list is supplied by the chosen Directory.

Reply:
On Linux master, I get
![image](https://user-images.githubusercontent.com/1549463/148696164-5bdf2832-6c10-4030-98fd-21c2f2849ab1.png)

I'm not sure where the (local) hostname is coming from.

Reply:
Sorry, does your comment imply a change to the wording here or not?

Reply:
Yes.  The text is wrong as it stands.

Reply:
OK I've re-written this in the new Custom-Directories.md file.